example/Python/src/ReflexGame/gui.py

```diff
@@ -0,0 +1,140 @@
+import multiprocessing
+from multiprocessing import connection
+import threading
+import sys
+import os
+os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"
+
+try:
+    import pygame
+except:
+     print("Import Error: Failed to import 'pygame'. Try 'pip3 install pygame'")
+     sys.exit(1)
+
+
+def start_gui():
```

Comment:
This should be something we could do in a reactor with federated Python once that's ready.